climate-monitoring-core/src/main/java/climatemonitoring/core/View.java

```diff
@@ -1,6 +1,46 @@
+/*
+
+Alessandro della Frattina 753073 VA
+Cristian Capiferri 752918 VA
+Francesco Lops 753175 VA
+Dariia Sniezhko 753057 VA
+
+*/
+
 package climatemonitoring.core;
 
-public class View {
+/**
+ * The View class manages the state of a view and handles rendering operations based on the configuration (headless or GUI)
+ * @author francescolops
+ * @version 1.0-SNAPSHOT
+ */
+public class View{
 
+	/**
+	 * Sets the view state
+	 * @param state
+	 */
+	public void setState(ViewState state){
+		
+		m_State = state;
+	}
 	
+	/**
+	 * Code that runs in headless configuration only
+	 * @param args
+	 */
+	public void onHeadlessRender(String args){
+		
+		m_State.onHeadlessRender(args);
+	}
+	
+	/**
+	 * Code that runs in GUI configuration only
+	 */
+	public void onGUIRender(){
+		
+		m_State.onGUIRender();
+	}
+
+	private ViewState m_State; 
```

Comment:
This field should not be capitalized, it should be written like this: `private ViewState m_state`